spirit/core/utils/markdown/block.py

```diff
@@ -47,6 +48,7 @@ class BlockGrammar(mistune.BlockGrammar):
         r'|youtu\.be/'
         r'|youtube\.com/embed/)'
         r'(?P<id>[a-zA-Z0-9_\-]{11})'
+        r'(&t=(?P<start_minutes>[0-9]+m)?(?P<start_seconds>[0-9]+s)?)?'
```

Comment:
I think both should match up to two digits ie: `(&t=(?P<start_minutes>[0-9]{1,2}m)?(?P<start_seconds>[0-9]{1,2}s)?)?`, also, matching *hours* is missing.